components/Stats/index.jsx

```diff
@@ -14,6 +15,12 @@ export default ({
 }) => {
   const router = useRouter();
   const activeCases = confirmed?.value - deaths?.value - recovered?.value;
+
+  const handleChange = event => {
+    const { value } = event;
+    localStorage.setItem(LAST_VISITED_COUNTRY, value);
```

Comment:
You should check if typeof `window` if it's not undefined as it's not available on SSR

Reply:
Are you sure ?
I thought that when the component mounted it means that you are on the browser ?

Reply:
https://www.gatsbyjs.org/docs/debugging-html-builds/

Reply:
They say use `componentDidMount`or `useEffect `

> Some of your code references “browser globals” like window or document. If this is your problem you should see an error above like “window is not defined”. To fix this, find the offending code and either a) check before calling the code if window is defined so the code doesn’t run while Gatsby is building (see code sample below) or b) if the code is in the render function of a React.js component, move that code into a componentDidMount lifecycle or into a useEffect hook, which ensures the code doesn’t run unless it’s in the browser.